config/value.go

```diff
@@ -128,14 +129,14 @@ func (v *atomicValue) Duration() (time.Duration, error) {
 }
 
 func (v *atomicValue) Scan(obj interface{}) error {
-	data, err := json.Marshal(v.Load())
+	data, err := stdjson.Marshal(v.Load())
 	if err != nil {
 		return err
 	}
 	if pb, ok := obj.(proto.Message); ok {
-		return protojson.Unmarshal(data, pb)
+		return json.UnmarshalOptions.Unmarshal(data, pb)
 	}
-	return json.Unmarshal(data, obj)
+	return stdjson.Unmarshal(data, obj)
```

Comment:
It is not possible to decode STD JSON directly, as fields may not match, It can be handled by two logic process.

Reply:
but proto json unmarshal should be can modify by user

Reply:
```go
func (v *atomicValue) Scan(obj interface{}) error {
	codec := encoding. GetCodec(json.Name)
	data, err := codec.Marshal(v.Load())
	if err != nil {
		return err
	}
	return codec.Unmarshal(data, obj)
}